### PR TITLE
feature/services-section: Section Services

### PR DESCRIPTION
**Pull request by ArleyAV**
SECTION SERVICES

• Mobile and desktop version were implemented.

NOTES
• I moved Section Services from style.css to home.css
• I moved some general styles from home.css to style.css, this way we can keep the general reset in one place.
• I replaced PNG icons for SVG instead.
• I deleted the media query code that I wrote because I hadn’t seen the media query setup that already exists, sorry my bad.
• I did refactor of the code based in Juanse’s feedback.

This is the way how it looks 

Desktop Version
image

Mobile Version
image

### CODE REVIEW

**Comment by jusk02** [comment]
@ArleyAV Hi, we do have a defined unique breakpoint at 960px, how ever this i have found is not easy to know as it is not very clear in our documentation, i will find a way to make this information visible.

Taking that into account the breakpoint structure you have, even being well implemented might not suit the project correctly, lets discuss this offline as doing it in comments would be redundant and complicated.

**Comment by jusk02** [comment]
Please add a screen shot of your work both on mobile and desktop viewports tho the PR description.

**Comment by jusk02** [comment]
Please use images in SVG instead of PNG, this way we can avoid having multi-size assets.

**Comment by jusk02** [comment]
Great work with the PR naming, Description, commit message and css structure, its super clean.

**Review by jhonatanrojasbastidas — COMMENTED** [review]
why the footer icons are inline block? ; in developt is fine.

**Comment by jhonatanrojasbastidas** [comment]
In the sketch cloud  in your section  i see a botton "cotiza tu estadia" and in your photo i dont  see it.

File: css/home.css [hunk 1]
```diff
@@ -99,8 +92,65 @@ body {
...
+  max-width: 218px;
+  text-align: center;
+  width: 100%;
+}
+
+/* END SERVICES */
+
+/* START MEDIA QUERIES */
+
 @media only screen and (min-width: 960px) {
```
Comment (jhonatanrojasbastidas) [inline css/home.css:135]: I have some doubts ; in the media query go only the selectors that change for the mobile or you can creates new selector for that. i think that you only create all in the mobile area and later you changes the specs for de desktop area but not create new ones. You can explain that, please?

**Review by jhonatanrojasbastidas — COMMENTED** [review]
Please answer the doubts i have .

File: css/style.css [hunk 2]
```diff
@@ -1,19 +1,43 @@
 /* General page styles */
 /* We should use a mobile first approach having out breack point at 768px */
-/*Comodities */
+
+/* START GENERAL STYLES */
+*,
+*::after,
+*::before {
```
Comment (hanna446) [inline css/style.css:7]: What are you doing in this line?

File: css/home.css (hunk 1 above)
Comment (jusk02) [reply css/home.css:135]: it is not the usual thing, but you can have a selector that does absolutely nothing in a viewport and does have some rules in other.

Comment (jhonatanrojasbastidas) [reply css/home.css:135]: ok.

**Comment by jusk02** [comment]
please remove all the `.DS_STORE` files, these are just trash data.

File: css/style.css (hunk 2 above)
Comment (josemariogutierrez) [reply css/style.css:7]: @ArleyAV Could you please answer this question?

Comment (ArleyAV) [reply css/style.css:7]: @hanna446 

Because this way we save code's lines. What I mean is we are applying the same rules to this selectors

Comment (josemariogutierrez) [reply css/style.css:7]: @hanna446 The * character makes reference to all elements in the dom, so this code is applying to all elements `box-sizing: border-box` which include padding and border in the element's total width and height. Also with `::before` and `::after` you are applying this to all pseudo-lements.

**Comment by ArleyAV** [comment]
> why the footer icons are inline block? ; in developt is fine.

@jhonatanrojasbastidas 
I defined a class for images and videos of full width, this mean every image must be fitted regarding the size of the container instead its own size, that's the reason every icon turned into blocks

**Comment by ArleyAV** [comment]
Hi! I followed your recomendations and comments. You can check again. I fixed the vertical icons in the footer.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (jusk02) [inline .gitignore:1]: this file still needs to be deleted

**Comment by jusk02** [comment]
Looks pretty good, just remove that last `.DS_store`

File: .gitignore (hunk 3 above)
Comment (josemariogutierrez) [reply .gitignore:1]: Ar you refering to the .gitignore file?

Comment (jusk02) [reply .gitignore:1]: ups, misread thought it was a ds_store

**Review by jusk02 — APPROVED** [review]

**Review by camigomezdev — APPROVED** [review]

File: .gitignore (hunk 3 above)
Comment (josemariogutierrez) [reply .gitignore:1]: Shame on you

**Review by josemariogutierrez — APPROVED** [review]

**Pull request merged**